app/assets/javascripts/osem-datatables.js

```diff
@@ -1,6 +1,6 @@
 $(function () {
   $(document).ready(function() {
-    $('.datatable').DataTable({
+    $('.data-table').DataTable({
```

Comment:
Here, using 'datatable' as the classname to identify and initialize datatables is trouble in the making.
The datatable library when initializing a table as a datatable adds 'dataTable' class to the table.So if someone were to initialize a table as datatable from someplace else, like done [here](https://github.com/openSUSE/osem/blob/master/app/views/admin/volunteers/_volunteers_table.html.haml#L24) , this would cause dataTable library to raise reinitialization error alert
![reinitialize-warning](https://cloud.githubusercontent.com/assets/7537349/17050634/9d5186dc-5011-11e6-95e1-d5ffd458a8d0.png)